### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by momih**
- Moved ReLU inside `GCNLayer`
- Added if condition for centroids, so max pool isn't called for prepool layers. Also moved permute outside `max_pool` 
- Added optional param for model.predict to return probs instead of logits
- Fixed `get_every_n` so that it doesn't miss `len(a) % n` samples

### CODE REVIEW

**Review by mweiss17 — APPROVED** [review]
LGTM
